src/bin/playbot.rs

```diff
@@ -250,9 +253,17 @@ fn main() {
                 shorten_key: bitly_key.clone(),
                 cache: cache.clone(),
             };
-            threads.push(thread::spawn(move || {
-                bot.main_loop();
-            }));
+            threads.push(thread::Builder::new()
+                                         .name(format!("{}@{}", nick, server_addr))
+                                         .spawn(move || {
+                if let Err(e) = catch_unwind(AssertUnwindSafe(|| bot.main_loop())) {
+                    error!("{:?}", e);
```

Comment:
Unfortunately this may not provide too much information as the `Debug` implementation for `Any` just prints "Any". This error should already have been printed, however, so it may not be necessary to print it again. Perhaps this could just print out a string that's easily greppable?